src/main/java/TicTacToe/Display.java

```diff
@@ -0,0 +1,65 @@
+package TicTacToe;
+
+import java.io.*;
+import java.util.List;
+
+public class Display {
+    public OutputStream outputStream;
+    private PrintStream printStream;
+    private BufferedReader inputStream;
+
+    public Display(PrintStream outputStream, InputStream inputStream) {
+        this.outputStream = outputStream;
+        this.printStream = new PrintStream(outputStream);
+        this.inputStream = new BufferedReader(new InputStreamReader(inputStream));
+    }
+
+    public void printGreeting() {
+        printStream.println("Welcome to TicTacToe");
+    }
+
+    public void printBoard(Board board) {
+        List<String> grid = board.getGridAsList();
+
+        String string = grid.get(0) + " | " + grid.get(1) + " | " + grid.get(2) + "\n"
+                + grid.get(3) + " | " + grid.get(4) + " | " + grid.get(5) + "\n"
```

Comment:
Are board and grid referring to the same thing? 

It feels that your code could describe more your intention. For example, `grid.get(3)` I believe that it will return the mark on the position three of the board but I'm not really sure, something like that would be easier to follow `board.getMarkOnPosition(3)` (that's just an example though :) )